meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -310,6 +313,12 @@ do_install() {
     install -d ${D}${sysconfdir}/udev/mount.blacklist.d
     echo ${MENDER_ROOTFS_PART_A} > ${D}${sysconfdir}/udev/mount.blacklist.d/mender
     echo ${MENDER_ROOTFS_PART_B} >> ${D}${sysconfdir}/udev/mount.blacklist.d/mender
+
+    # install the D-Bus interface file
+    if [ -f ${B}/src/${GO_IMPORT}/Documentation/io.mender.Authentication1.xml ]; then
+        install -d ${D}${datadir}/dbus-1/interface
+        install -m 0644 ${B}/src/${GO_IMPORT}/Documentation/*.xml ${D}${datadir}/dbus-1/interface
```

Comment:
Just in case we eventually add a non-DBus xml file in the folder, I would suggest:

```suggestion
        install -m 0644 ${B}/src/${GO_IMPORT}/Documentation/io.mender.*.xml ${D}${datadir}/dbus-1/interface
```